### PR TITLE
Hpa beta2

### PR DESCRIPTION
**Pull request by libuweber**
# Pull Request Template
use autoscaling/v2beta2 if k8s version is >= 1.12. 

With v2beta2, custom metrics can be used.

### CODE REVIEW

File: charts/microgateway/values.yaml [hunk 1]
```diff
@@ -329,9 +329,11 @@ hpa:
   minReplicas: 1
   resource:
     # hpa.resource.memory -- Average Microgateway Memory consumption to scale up/down.<br><br>
-    # :exclamation: Update this setting accordingly to `resources.limits.memory`.
-    memory: 2Gi
-    # hpa.resource.cpu -- Average Microgateway CPU consumption in percentage to scale up/down.
+    # :exclamation: Update this setting according to `resources.limits.memory`.
+    memory: 3Gi
```
Comment (dieti80) [inline charts/microgateway/values.yaml:335]: Why did you change to 3Gi?

**Review by dieti80 — COMMENTED** [review]
The changes look fine. But the custom metrics are not yet implemented, right?

**Comment by libuweber** [comment]
> The changes look fine. But the custom metrics are not yet implemented, right?

correct.

**Review by dieti80 — APPROVED** [review]
Looks good
